app/controllers/trainees/lead_schools_controller.rb

```diff
@@ -68,7 +68,7 @@ def query
     end
 
     def index_or_edit_page
-      @lead_school_form.index_page_radio_buttons? ? :index : :edit
+      @lead_school_form.search_results_found? ? :index : :edit
```

Comment:
I think you'll need to change this in the employing schools controller too or you'll get an undefined method error